snuba/datasets/plans/single_storage.py

```diff
@@ -147,7 +147,7 @@ def build_and_rank_plans(
                 plan_query_processors=[],
                 db_query_processors=db_query_processors,
                 storage_set_key=self.__storage.get_storage_set_key(),
-                execution_strategy=SimpleQueryPlanExecutionStrategy(
+                execution_strategy=SimpleQueryPlanExecutionStrategy(  # type: ignore
```

Comment:
Mind adding a comment with the reference to the mypy bug

Reply:
My mistake, this should be fixed with the update. Reverting this line.